runners/flink/src/main/java/org/apache/beam/runners/flink/translation/functions/FlinkPartialReduceFunction.java

```diff
@@ -98,11 +98,10 @@ public void combine(
     if (groupedByWindow) {
       reduceRunner = new SingleWindowFlinkCombineRunner<>();
     } else {
-      if (windowingStrategy.needsMerge()
-          && !windowingStrategy.getWindowFn().windowCoder().equals(IntervalWindow.getCoder())) {
-        reduceRunner = new HashingFlinkCombineRunner<>();
-      } else {
+      if (windowingStrategy.needsMerge() && windowingStrategy.getWindowFn() instanceof Sessions) {
```

Comment:
is `SingleWindowFlinkCombineRunner` actually what is used for non-merging?

Reply:
I think so.